### DISCUSSION TITLE
save changes in kitty config file

### DISCUSSION THREAD

**Discussion by keyd08** [kovidgoyal/kitty#7415] [General]
Hi there! Can someone help me understand how to simply save and exit after a modification made in the config kitty file? I press ctrl O and then when it shows if I want to save into the kitty config file I just press enter but doesn't work .... it's a real headache .... thanks a lot!

**Comment by Luflosi** [comment]
This depends on the editor you're using. From what you're describing, it sounds like you're using `nano`. To exit `nano`, press <kbd>ctrl</kbd>+<kbd>x</kbd>. If there are any unsaved modifications, `nano` will ask if you want to save them, which you can confirm with (probably, depending on your language settings) Y and then <kbd>enter</kbd>.
